docs/health.md

```diff
@@ -0,0 +1,25 @@
+# Health service
+The Health Service is a component at the service layer of the BDK which covers the Health Service part of the [REST API documentation](https://developers.symphony.com/restapi/reference).
+More precisely:
+* [Health check](https://developers.symphony.com/restapi/reference/health-check-v3)
+* [Health check extended](https://developers.symphony.com/restapi/reference/health-check-extended-v3)
+* [Agent info](https://developers.symphony.com/restapi/reference/agent-info-v1)
+
+
+## How to use
+The central component for the Health Service is the `UserService` class, it exposes the service APIs endpoints mentioned above.
+The service is accessible from the`SymphonyBdk` object by calling the `users()` method:
```

Comment:
_Health Service is the `**UserService**` class_,   should it be **HealthService** ?  and **_health()_** for the next line maybe